### PR TITLE
Use pre-commit in contributing section

### PR DESCRIPTION
**Pull request by fsimonis**
The contributing section mentions a lot of checks, which are now automated using `pre-commit`.
This PR replaces the majority of the section with "please use pre-commit" :wink:

Related to https://github.com/precice/tutorials/pull/464

### CODE REVIEW

File: pages/community/community-contribute-to-precice.md [hunk 1]
```diff
@@ -114,45 +114,15 @@ Your case may already fit into one of the existing tutorials. If not, feel free
 - Watchpoint names should be describing the point, not be a generic name.
 - Images need to be named as `tutorials-<tutorial>-<image>.png` to be correctly displayed on the website. You can then refer to them as `![title](images/tutorials-<tutorial>-<image>.png)`. Subdirectories are not allowed.
 
-### Optional: Help us with some checks
```
Comment (fsimonis) [inline pages/community/community-contribute-to-precice.md]: Is this change a bit too authoritative?

Comment (MakisH) [reply pages/community/community-contribute-to-precice.md]: Yes, there has already recently been intense discussions regarding these guidelines, let's not intensify this, please. :upside_down_face:

File: pages/community/community-contribute-to-precice.md [hunk 2]
```diff
@@ -114,45 +114,15 @@ Your case may already fit into one of the existing tutorials. If not, feel free
...
-
-  ```bash
-  npm install -g markdownlint-cli
-  # See also https://stackoverflow.com/a/54170648/2254346
-  markdownlint .
-  ```
-
-We automate many of these checks with [GitHub actions](https://github.com/features/actions), which you will see running at the bottom of each pull request. To avoid pushing and waiting for the actions to run while you develop, you can alternatively install [act](https://github.com/nektos/act) to execute all or specific workflows locally, running `act` or `act -j <job_name>`. It requires [Docker](https://www.docker.com/) and you can get the latest binary from the [act releases](https://github.com/nektos/act/releases/latest).
-</details>
+We automate many checks with [GitHub actions](https://github.com/features/actions), which you will see running at the bottom of each pull request. Using `pre-commit` yourself saves you some unnecessary trouble.
```
Comment (MakisH) [inline pages/community/community-contribute-to-precice.md:125]: Do we really already include 

- checking for commented-out code
- gitignore
- shellcheck
- unused elements in the config

in the pre-commit?

Don't forget that this list also serves as a checklist for reviewers.

**Review by MakisH — CHANGES_REQUESTED** [review]

File: pages/community/community-contribute-to-precice.md (hunk 2 above)
Comment (uekerman) [reply pages/community/community-contribute-to-precice.md:125]: "unused elements in the config" is sth we cannot do yet (you mean the preCICE config, right?), but sth we promised for preECO.

Comment (fsimonis) [reply pages/community/community-contribute-to-precice.md:125]: Ah shellcheck is probably still missing. Will add that next week to the tutorials.

The gitignore and cleaning up commented code are still mentioned below. Its hard to see in all the diff noise.

Comment (fsimonis) [reply pages/community/community-contribute-to-precice.md:125]: Added it.

File: pdf-docs.sh
Comment (MakisH) [inline pdf-docs.sh:1]: Is this related?

**Review by MakisH — APPROVED** [review]

File: pages/community/community-contribute-to-precice.md (hunk 2 above)
Comment (fsimonis) [reply pages/community/community-contribute-to-precice.md:125]: Added shellcheck to the tutorials
